packages/story-editor/src/elements/shape/icon.js

```diff
@@ -55,21 +63,18 @@ function ShapeLayerIcon({
   if (isDefaultBackground) {
     backgroundColor = currentPage.backgroundColor;
   }
+
   return (
     <Container>
-      <ShapePreview
-        style={{
-          clipPath: `url(#${maskId})`,
-        }}
-        width={PREVIEW_SIZE * maskDef.ratio}
-        height={PREVIEW_SIZE}
-        backgroundColor={backgroundColor}
-      >
+      <ShapePreview maskId={maskId} backgroundColor={backgroundColor}>
         <svg width={0} height={0}>
           <defs>
             <clipPath
               id={maskId}
-              transform={`scale(1 ${maskDef.ratio})`}
+              // Bring the path scale down a bit from 1
+              // so that we can make sure the entire SVG path is visible when the mask ratio is > 1
+              // this is important for Firefox's interpretation of clip paths
+              transform={`scale(1 0.9)`}
```

Comment:
Don't think we need string interpolation here?
```suggestion
              transform="scale(1 0.9)"
```

Reply:
yup! tried this out and it worked and i didn't clean up. on it! 👍

Reply:
Oh, gtk!